runtime/interpreter/src/interpreter.cpp

```diff
@@ -387,12 +436,149 @@ int handleIntrinsic(ShrimpVM *vm)
     return dispatch_table[getOpcode(vm->pc())](vm);
 }
 
+int handleCall0arg(ShrimpVM *vm)
+{
+    Instr<InstrOpcode::CALL_0ARG> instr {vm->pc()};
+
+    auto func_id = instr.getFuncId();
+
+    LOG_INFO(instr.toString(), vm->getLogLevel());
+
+    vm->stack().push(Frame {std::make_shared<RuntimeFunc>(vm->resolveFunc(func_id))});
+
+    auto &frame = vm->currFrame();
+
+    ByteOffset offset = frame.getOffsetToFunc();
+    frame.setRetPc(vm->pc() + instr.getByteSize());
+    vm->pc() = vm->getPcFromStart(offset);
+
+    return dispatch_table[getOpcode(vm->pc())](vm);
+}
+
+int handleCall1arg(ShrimpVM *vm)
+{
+    Instr<InstrOpcode::CALL_1ARG> instr {vm->pc()};
+
+    auto func_id = instr.getFuncId();
+    auto func_0arg_idx = instr.getFuncArg0();
+
+    LOG_INFO(instr.toString(), vm->getLogLevel());
+
+    auto &prev_frame = vm->currFrame();
+    vm->stack().push(Frame {std::make_shared<RuntimeFunc>(vm->resolveFunc(func_id))});
+    auto &frame = vm->currFrame();
+
+    auto func_0arg = prev_frame.getReg(func_0arg_idx).getValue();
+    frame.setReg(func_0arg, 255);
```

Comment:
Replace magic numbers with consts?

Reply:
Assigned it to you, must be replaced after your changes of argument register allocation